cogs/draw_image_generation.py

```diff
@@ -6,20 +6,19 @@
 
 import discord
 from PIL import Image
-from discord.ext import commands
+from pycord.multicog import add_to_group
 
 
 # We don't use the converser cog here because we want to be able to redo for the last images and text prompts at the same time
 from models.env_service_model import EnvService
 from models.user_model import RedoUser
-from models.check_model import Check
 
 redo_users = {}
 users_to_interactions = {}
 ALLOWED_GUILDS = EnvService.get_allowed_guilds()
 
 
-class DrawDallEService(commands.Cog, name="DrawDallEService"):
+class DrawDallEService(discord.Cog, name="DrawDallEService"):

```

Comment:
I want to confirm the difference here between commands.Cog and discord.Cog, it seems I've been using the old version (commands.Cog), right?

Reply:
Afaik discord.Cogs and discord.ext.commands.Cogs works in the same way. But since we're not using any discord.ext functionality we don't really need to keep it imported and can thus just change commands.Cog to discord.Cog. Not really an important change either way it goes